src/absenceTypes/types.ts

```diff
@@ -2,43 +2,43 @@ export type AbsenceType = {
   /**
    * Id of the absence.
    */
-  id?: number;
+  id: number;
```

Comment:
this would mean on create we have to sent id. Maybe add same we have in timetrackings, to have different types for read, create, update?